src/test/java/io/vertx/test/core/MetricsTest.java

```diff
@@ -493,15 +504,17 @@ public void testHttpConnect() {
         assertFalse(serverMetric.socket.connected.get());
         assertEquals(5, serverMetric.socket.bytesRead.get());
         assertEquals(5, serverMetric.socket.bytesWritten.get());
+        assertEquals(serverMetric.socket.remoteAddress.host(), serverMetric.socket.remoteName);
```

Comment:
I guess this is ok to test the server metric side, provided Netty does not resolve the address internally.